WorkshopToolCommon/Options/PublishVerbBase.cs

```diff
@@ -25,6 +25,9 @@ public abstract class PublishVerbBase : UGCOptionBase
         [Option("dependencies", HelpText = "Specify dependencies to other mods. Use 0 to remove all.", MetaValue = "<id>")]
         public IEnumerable<string> Dependencies { get; set; }
 
+        [Option("discord-webhook", HelpText = "A link to a webhook that will publish update notes", MetaValue = "<url>")]
```

Comment:
This new code isn't released yet, so I want to make sure the understanding is clear.
PublishVerbBase matches both the push verb (uploading a mod), and changing metadata of a mod, without a content change (such as adding tags).

**Is the intention of this to support discord web hooks even if the content of the mod hasn't changed?**

Reply:
Ah no- I will move that into UploadVerb as it should only POST to the webhook when a file was changed